root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1787,3 +1776,118 @@ test('Cleanup', function (t) {
 
     t.end();
 });
+
+test('URL cleanup component: auto-select, clean-up, and validation', {}, function (t) {
+    const test_data = [
+        // AllMusic
+        {
+            i_url: 'http://www.allmusic.com/album/here-comes-the-sun-mw0002303439/releases',
+            i_ent: 'release_group',
+            match: 'allmusic',
+            clean: 'http://www.allmusic.com/album/mw0002303439',
```

Comment:
Alternatively, use longer/more descriptive property names that are self-explanatory (`input_url`, `output_relationship`, etc.). I like that they're all 5 chars but you can always use whitespace to align them. :)